app/src/main/res/values/strings.xml

```diff
@@ -217,4 +218,10 @@
     <string name="contribute_code_to_this_project_anybody_welcome">Contribute code to the project. Everybody is welcome to do so, including newcomers</string>
     <string name="project_team">Project Team</string>
     <string name="contributors_public_info">Show contributer info. Provide it to be shown here on an opt-in basis after contributing</string>
+
+    <!-- File Info Dialog -->
+    <string name="location">Location</string>
```

Comment:
apart location i think everything is already available as string, use them instead. OK -> android.R.string.ok